### PR TITLE
Logic cleanup

### PR DESCRIPTION
**Pull request by Roleren**

### CODE REVIEW

File: tests/testthat/test_cage.R [hunk 1]
```diff
@@ -22,7 +22,7 @@ start(cageEqualStart)[3] <- start(cageEqualStart)[3] - 1
 
 test_that("reassignTSSbyCage picks best one max peak of several", {
   # second granges have higher score, and both are within frame, then max one should be picked
-  test_result <- reassignTSSbyCage(fiveUTRs[1], cage = cageEqualStart, cds = cds)
+  test_result <- suppressWarnings(reassignTSSbyCage(fiveUTRs[1], cage = cageEqualStart, cds = cds))
```
Comment (JokingHero) [inline tests/testthat/test_cage.R]: no, this is not the correct fix for min or max

File: tests/testthat/test_features.R [hunk 2]
```diff
@@ -371,7 +371,7 @@ test_that("computeFeatures works as intended", {
   expect_equal(nrow(dt), 4)
 
   # test from example table in orfik
-  dt <- computeFeatures(grl = grl,orfFeatures = T, RFP = RFP7, RNA = RNAGAlign,
+  dt <- computeFeaturesSpecial(grl = grl,orfFeatures = T, RFP = RFP7, RNA = RNAGAlign,
```
Comment (JokingHero) [inline tests/testthat/test_features.R]: name of the special will definitely not be special

File: tests/testthat/test_cage.R (hunk 1 above)
Comment (Roleren) [reply tests/testthat/test_cage.R]: This is not for max min, this a bug in GenomicFeatures which will be fixed in bioC 3.7. We can remove it then.

File: tests/testthat/test_features.R (hunk 2 above)
Comment (Roleren) [reply tests/testthat/test_features.R]: What about computeFeaturesCage ?
I will try to make the txdb maker function now, and see what we can do to remove this function.

File: tests/testthat/test_cage.R (hunk 1 above)
Comment (JokingHero) [reply tests/testthat/test_cage.R]: I thought you are working with the devel version of bioC 3.7 as you should if you want to push it to this bioc version! if this is true we dont need suppressWarnings at all and it all should be working just fine

File: tests/testthat/test_features.R (hunk 2 above)
Comment (JokingHero) [reply tests/testthat/test_features.R]: Mixing logic of reassigning start codons with calculating features just can not go together as those are completely two separate problems.

File: tests/testthat/test_cage.R (hunk 1 above)
Comment (Roleren) [reply tests/testthat/test_cage.R]: I'm using 3.6, should I change now ?

**Pull request merged**